pkg/capture/capture_manager.go

```diff
@@ -233,6 +241,29 @@ func (cm *Manager) Update(ifaces config.Ifaces, returnChan chan TaggedAggFlowMap
 
 }
 
+// StateAll returns the states of all managed Capture instances (without touching
```

Comment:
Only here temporarily as a reminder for our upcoming refactor. Might not be needed.